plugin/types.go

```diff
@@ -195,11 +195,8 @@ type Readable interface {
 	Open(context.Context) (SizedReader, error)
 }
 
-// Deletable is an entry that can be deleted. Entries that implement Delete
-// should ensure that it and all its children are removed. If the entry has
-// any dependencies that need to be deleted, then Delete should return an
-// error.
+// Deletable is an entry that can be deleted.
```

Comment:
Why'd you remove the description? We also need to describe what the boolean return value represents.

Reply:
I was trying to place the plugin-author specific action info in the external plugin docs and leave the comments here to core-plugin specific subtleties. I guess it wouldn't hurt to include a brief description.